src/07-registers/type-safe-manipulation.md

```diff
@@ -175,28 +192,38 @@ did!
 
 [LTO]: https://en.wikipedia.org/wiki/Interprocedural_optimization
 
+Use `cargo objdump` to grab the assembler code to `release.txt`:
 ``` console
-$ cargo objdump --bin registers --release -- -d --no-show-raw-insn --print-imm-hex
-registers:      file format ELF32-arm-little
-
-Disassembly of section .text:
-main:
- 8000188:       bl      #0x22
- 800018c:       movw    r0, #0x1018
- 8000190:       mov.w   r1, #0x200
- 8000194:       movt    r0, #0x4800
- 8000198:       str     r1, [r0]
- 800019a:       mov.w   r1, #0x800
- 800019e:       str     r1, [r0]
- 80001a0:       mov.w   r1, #0x2000000
- 80001a4:       str     r1, [r0]
- 80001a6:       mov.w   r1, #0x8000000
- 80001aa:       str     r1, [r0]
- 80001ac:       b       #-0x4 <main+0x24>
+cargo objdump --bin registers --release -- -d --no-show-raw-insn --print-imm-hex > release.txt
```

Comment:
```suggestion
$ cargo objdump --bin registers --release -- -d --no-show-raw-insn --print-imm-hex > release.txt
```

Reply:
I purposely didn't add the `$` as this makes the `copy` button more useful.